lib/cache/access_list.go

```diff
@@ -260,10 +260,10 @@ func (c *Cache) ListAllAccessListMembers(ctx context.Context, pageSize int, page
 		defaultPageSize: 200,
 		upstreamList:    c.Config.AccessLists.ListAllAccessListMembers,
 		nextToken: func(t *accesslist.AccessListMember) string {
-			return t.GetMetadata().Name
+			return t.Spec.AccessList + "/" + t.GetMetadata().Name
```

Comment:
The member key `<list-name>/<member-name>` is used in multiple places in the cache, and looks like part of the issue it was originally missed, can this be factored out into a helper method?